documentation/Contributions-and-ApprovalProcess.md

```diff
@@ -0,0 +1,41 @@
+# Contributions and Approval Process
+
+
+We propose creating intent based subgroups for APIs, identified for now as different subdirectories within the EdgeCloud repository. (Later these could be own repositories which offer cleaner separation)
+For eg.
+
+* Edge-discovery
+* Application-LCM
+
+<br>
+**Branching and Approaval process**
+For the API/APIs where multiple contributors showcase capability overlap, the core contributors in a smaller round can agree on the first feature(API) for contribution. If for eg.  feature-1 (which has an overlap) within the intent group AppLCM is planned to be contributed, an issue gets created to mention the brief details such as agreed API name, file name, contributor list etc. This could be contributed as AppLCM-F1.MD file and can include these overview details.
```

Comment:
This should align with the ongoing CAMARA 'fork and pull' proposal so this part may have to be tweaked.